qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/diagnostics/QuteDiagnosticsInExpressionWithNamespaceTest.java

```diff
@@ -104,14 +104,15 @@ public void invalidInjectNamespaceInMemberPart() {
 		testDiagnosticsFor(template,
 				d(0, 13, 0, 17, QuteErrorCode.UnknownProperty,
 						"`XXXX` cannot be resolved or is not a field of `java.lang.String` Java type.",
-						new JavaBaseTypeOfPartData("java.lang.String"), DiagnosticSeverity.Error));
+						new JavaBaseTypeOfPartData("java.lang.String"), //
+						DiagnosticSeverity.Error));
 
 		template = "{inject:bean.XXXX()}";
 		testDiagnosticsFor(template,
 				d(0, 13, 0, 17, QuteErrorCode.UnknownMethod,
 						"`XXXX` cannot be resolved or is not a method of `java.lang.String` Java type.",
+						new JavaBaseTypeOfPartData("java.lang.String"), //
 						DiagnosticSeverity.Error));

```

Comment:
I think it could be intéressant to add a testCodeqctionFor here to test method coming from namepscze